daos/author.go

```diff
@@ -2,14 +2,63 @@ package daos
 
 import (
 	"context"
+	"database/sql"
+	"fmt"
 	"go-template/models"
+
+	"github.com/volatiletech/sqlboiler/v4/boil"
+
+	"github.com/volatiletech/sqlboiler/v4/queries/qm"
 )
 
-func FindAuthorByEmail(email string, ctx context.Context) (models.Author, error) {
-	return models.Author{}, nil
+func FindAuthorByEmail(email string, ctx context.Context) (*models.Author, error) {
+	contextExecutor := getContextExecutor(nil)
+	return models.Authors(qm.Where(fmt.Sprintf("%s=?", models.AuthorColumns.Token), email)).One(ctx, contextExecutor)
```

Comment:
is `models.AuthorColumns.Token` column an email column? looks like you need to query by email here